lib/rubocop/cop/style/rescue_standard_error.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for rescuing `StandardError`. There are two supported
+      # styles `implicit` and `explicit`. This cop will not register an offense
+      # if any error other than `StandardError` is specified.
+      #
+      # @example
+      #
+      #   # good
+      #   begin
+      #     foo
+      #   rescue OtherError
+      #     bar
+      #   end
+      #
+      #   # good
+      #   begin
+      #     foo
+      #   rescue StandardError, OtherError
```

Comment:
Isn't this example shadowing `OtherError`? 🙂

Reply:
Not necessarily. Errors don't have to inherit from `StandardError`. If `OtherError` inherit from `Exception`, then there wouldn't be any shadowing going on. I can change `OtherError` to `SecurityError`, or another error, that Ruby defines and does not inherit from `StandardError`.

```ruby
[4] pry(main)> SecurityError.ancestors
=> [SecurityError, Exception, Object, PP::ObjectMixin, Kernel, BasicObject]
```

Reply:
We also have a cop that says "don't inherit from `Exception`". Ideally our suggestions shouldn't conflict with other cops when possible. 🙂